content/en/docs/concepts/_index.md

```diff
@@ -1,17 +1,17 @@
 ---
 title: Concepts
 main_menu: true
-content_template: templates/concept
+# content_template: templates/concept
```

Comment:
I'm ambivalent about whether to keep these lines or remove them, but it's not a blocker for this PR.

I think it's worth reviewing and possibly iterating later, but this seems like a reasonably conservative approach for now. If they prove to be crufty after 90+ days, it should be trivial to remove them later.

Reply:
I agree. I was hesitant to remove entirely.

Reply:
Some more thoughts on this. I don't want to leave a variable commented out in the front matter. I think it makes sense to either:
1. Remove the content_template variable if the `templates` files are no longer needed. 
2. Change the content_template variable/value to: `content: concept`, `content: task`, `content: tutorial`. Leave some information about the page type so that there is flexibility in the future.

Reply:
> `content: concept` (etc)

I like the idea of this: it makes it easier to add a future lint for (for example) task-style content in the concepts section.

Reply:
@kbhawkey _if_ I wanted to add some JSON-LD metadata to declare, in the rendered page, “this page describes a concept” what approach would you recommend? A render hook?

I'm thinking of starting out with a no-op render hook that checks for a valid value of either `content_template` or `content` and doesn't otherwise change the pages. However, I've never tried using Hugo render hooks.

Reply:
@sftim, Seems interesting. Do you need a render hook? The metadata appears in the header?

Reply:
Thinking,
content_type: tasks vs. content_type:task?
The directory structure uses the plural and the `card` dict in the front matter uses the plural for `name`.
```
card:
  name: tasks
  weight: 10
```